platform/ios/platform/ios/src/MGLMapView+OpenGL.mm

```diff
@@ -4,11 +4,7 @@
 
 #include <mbgl/gl/renderable_resource.hpp>
```

Comment:
It seems like this file should be renamed

Reply:
The code is still pretty much `OpenGL` 🤔 it's really just the overriding header files and bundled `MetalANGLE` doing the transition magic. I think there's a plan to rewrite the SDK in native Metal, that would be a good opportunity for changes – till that time, the SDK is formally still OpenGL.